library/custom_template/custom_template.php

```diff
@@ -50,7 +50,7 @@ function listitemCode($strDisp, $strInsert)
 ?>
 <html>
```

Comment:
This is showing ' TypeError: $(...).sortable is not a function'

![image](https://user-images.githubusercontent.com/278968/75623596-5ccabd00-5b60-11ea-961b-1cdcefc3f056.png)

(I'll show a video later on how to use Nation Note, so you can recreate/fix this bug)

Reply:
can't believe i am doing this for two days and have't found a optimal solution :cry:

Reply:
we may need to call in super Jerry (aka @sjpadgett ) :)

![tenor](https://user-images.githubusercontent.com/278968/76137320-4dc88c80-5ff0-11ea-83f1-c6b381dfa081.gif)

Reply:
@bradymiller i did created a workaround using interact.js but not getting the real feel.

Reply:
> I can't replace the jquery-ui function till i don't what is expected of him.

@im-Amitto 
There will be several places in codebase where jquery-ui was included to keep dialog from having to include. This is one of them i'm pretty sure.